server/events/external_team_allowlist_checker.go

```diff
@@ -23,20 +23,32 @@ func (checker *ExternalTeamAllowlistChecker) IsCommandAllowedForTeam(ctx models.
 	cmd := checker.buildCommandString(ctx, []string{team}, command)
 	out, err := checker.ExternalTeamAllowlistRunner.Run(ctx, "sh", "-c", cmd)
 	if err != nil {
+		ctx.Log.Err("Command '%s' error '%s'", cmd, err)
 		return false
 	}
 
-	return checker.checkOutputResults(out)
+	outputResults := checker.checkOutputResults(out)
+	if !outputResults {
+		ctx.Log.Info("command '%s' returns '%s'", cmd, out)
```

Comment:
I'm not convinced that these should be `Info` messages. I think `Debug` would be better, so they only display when an Atlantis command is run with the `-verbose` flag . Same below.

Your examples in the description aren't useful, as they are made up. Please can you provide real world usage messages.